test/test-not-readable-nor-writable-on-read-error.c

```diff
@@ -0,0 +1,104 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
libuv contributors

Reply:
`Copyright the libuv project contributors. All rights reserved.` ? That's used on 5 other files. All of the rest have this header.

Reply:
Yes, that's the one. New files get the new copyright boilerplate.